src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -10,8 +10,11 @@ import org.spongycastle.math.ec.{ECAlgorithms, ECPoint, ECCurve}
 import org.spongycastle.util.BigIntegers._
 
 object ECDSASignature {
+
   val RAndSLength = 32
```

Comment:
I added that variable when adopting ScalaStyle, it is used to mean the length of both `r` and `s` from the `ECDSASignature` class. I agree that is not a very good name, maybe we could split it in 2 or come up with a better name.